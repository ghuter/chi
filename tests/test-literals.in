"Hello"
"123"
",;+-0[]{}()"
