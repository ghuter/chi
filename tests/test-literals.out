LITERAL(4)
"Hello"
NEWLINE(6)
LITERAL(4)
"123"
NEWLINE(6)
LITERAL(4)
",;+-0[]{}()"
EOI(1)
